rtflib/tests/test_rtflib.py

```diff
@@ -19,6 +19,26 @@
 }
 """
 
+HELLO_WORLD_MULTI = """
```

Comment:
Can you remove all these variables in the test case (`HELLO_WORLD`, `HELLO_WORLD_MULTI`, `TABLE`, etc.)?

I replaced them with rtf files so they are unused now.